### DISCUSSION TITLE
在使用 AdapterUniapp 适配器的时候, 怎么使用 Token认证拦截器

### DISCUSSION THREAD

**Discussion by dodu2014** [alovajs/alova#452] [Q&A]
如题: 我使用的是 AdapterUniapp 适配器

![image](https://github.com/user-attachments/assets/53099dc2-cecf-40fa-bced-126c1a8df774)

![image](https://github.com/user-attachments/assets/8013038c-99dd-4846-add3-b3f1719cf3f1)

报告类型不一致

**Comment by MeetinaXD** [comment]
类型不一致的报错提供一下

**Comment by JOU-amjs** [comment]
```ts
import VueHook from 'alova/vue';
import { uniappRequestAdapter } from '@alova/adapter-uniapp';

createServerTokenAuthentication<typeof VueHook, typeof uniappRequestAdapter>({ ... })
```
